_docs/technical/plutus/2017-01-01-introduction.md

```diff
@@ -1,8 +1,8 @@
 ---
 layout: default

```

Comment:
Since they are on the left-hand side of the diff as well, I believe they were there before. Check this out: https://github.com/input-output-hk/cardano-docs.iohk.io/blame/master/_docs/cardano/plutus/2017-01-01-introduction.md